helmfile.d/helmfile-60.teams.yaml

```diff
@@ -187,7 +186,31 @@ releases:
           - azure{{ end }}
           {{- if $team | get "stack.sitespeed" false }}
           - sitespeed{{ end }}
-
   {{- end }}
   {{- end }}
+  {{- if hasKey $team "jobs" }}
```

Comment:
Before you start ranting about this check @Morriz, it introduces an error without it if you run `otomi template -l group=jobs`. If there is another solution feel free to modify it but this works for now.